### DISCUSSION TITLE
More data from Cliff Sensors / IR

### DISCUSSION THREAD

**Discussion by Kola50011** [iRobotEducation/create3_docs#319] [Questions] [answered]
Hey! 
I am working on a project, where we would like to detect a black line on the floor using the 4 cliff sensors. Sadly, this information not published by the ROS API. Is there any way to access such information, other than the _/hazard_detection_ topic? We  would greatly appreciate something similar to the _/ir_intensity_ topic.

**Comment by shamlian** [comment, marked as answer]
I think the answer right now is "not immediately, but it's something we can schedule," so I've created a feature request here: https://github.com/iRobotEducation/create3_docs/issues/320
